spec/presenters/coronavirus/sub_section_json_presenter_spec.rb

```diff
@@ -158,6 +154,142 @@
     end
   end
 
+  describe "#sub_sections" do
```

Comment:
It looks like there is already a `#sub_sections` in this file which this should be merged into.